tests/BandHQStats.test.jsx

```
@@ -1,37 +1,44 @@
+  ProgressBar: () => <div data-testid='progress-bar' />,
+  Panel: ({ title, children }) => (
+    <div data-testid='panel'>
+      <div data-testid='panel-title'>{title}</div>
+      {children}
+    </div>
+  ),
+  Tooltip: ({ children }) => <div data-testid='tooltip'>{children}</div>,
+  SettingsPanel: () => <div />,
+  VolumeSlider: () => <div />,
+  Modal: () => <div />,
+  ActionButton: () => <button />
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)